apps/frontend/src/pages/Account.svelte

```diff
@@ -40,85 +40,38 @@
     const icon = jazzicon(40, hashdisplayName(displayName));
     displayNameContainer.prepend(icon);
   })();
-
-  function downloadKeys() {
-    const key = packKey($SiteKeyStore);
-    const fileToSave = new Blob([JSON.stringify(key)], {
-      type: 'application/json',
-    });
-    saveAs(fileToSave, 'baf-wallet.json');
-  }
 </script>
 
 <Layout>
-  <Card classExtra="flex flex-col container mx-auto">
-    {#if initNearAccount}
-      <h1 class="pb-6 text-4xl text-center">Account</h1>
-      <div
-        bind:this={displayNameContainer}
-        class="flex flex-row items-center justify-center pb-6"
+  {#if noChainInit}
+    <h1>Hi there!</h1>
+    <p>
+      It looks like you have not setup your account on any of the Blockchains
+      which we support.
+    </p>
+    <p>
+      Please see initialize your <a href="/settings" use:link>account's page</a>
+    </p>
+  {:else}
+    <h1>Account</h1>
+    <!-- <div>
```

Comment:
should commented code be removed or are you planning on doing something with this?

Reply:
Woops again, I think that was there before, I'll j remoe for now